assets/app/view/game/game_meta.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module View
+  class GameMeta < Snabberb::Component
```

Comment:
Since this is shown on the site as "Info", could we call the class and file "Info" as well? Easier to find the relevant class that way 🙂

Reply:
This component is a shared sub-component of the "Game Info" component that already exists.

Reply:
aren't you missing a module Game ?